notebooks/124-hugging-face-hub/124-hugging-face-hub.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>it is also important to show or mention save_pretrained method that allow to save model on disk for avoid next time conversion.</p><p> </p><p>Also good to mention that Optimum Intel accepts already converted models too and the openvino models can be downloaded directly from hf hub.</p><p>they can be found by this link <a href="https://huggingface.co/models?library=openvino&amp;sort=trending" target="_blank">https://huggingface.co/models?library=openvino&amp;sort=trending</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>done</p>